pkg/yang/find.go

```diff
@@ -63,8 +58,6 @@ func FindGrouping(n Node, name string, seen map[string]bool) *Grouping {
 		v = e.FieldByName("Import")
 		if v.IsValid() {
 			for _, i := range v.Interface().([]*Import) {
-				// TODO(borman): This is arguably wrong.  We
-				// need to do prefix matching.
 				pname := strings.TrimPrefix(name, i.Prefix.Name+":")
```

Comment:
So just to be clear, we're now saying this is not wrong - because when we do any `FindXXX` then we'll support using the prefix within the context of the module that we found it?

Reply:
I think this wasn't wrong in the first place because `name` MUST be either prefixed with the same prefix from a particular import statement, or otherwise it's not from the module. Thus we trim it before searching. Otherwise we skip it (`if pname == name`).

Reply:
I'll add a comment, I had to re-read the code to figure this out again.